test-e2e/ts/step_definitions/custom.ts

```diff
@@ -77,4 +86,9 @@ When('write {string} to {value} value', async function(value: string, key: Memor
 
 When('I expect {value} {validation} {value}', async function(value1: MemoryValue, validate: Validation, value2: MemoryValue) {
     validate(value1.value(), value2.value());
-});
\ No newline at end of file
+});
+
+When('I click {string} and verify {string}', Template((locator: string, expected: string) => `
```

Comment:
how will it look in the report? as only 1 step or with internal steps also?

Reply:
as one step, it will report nested steps in error message if any

Reply:
got it, thanks